extensions/amp-brid-player/0.1/amp-brid-player.js

```diff
@@ -235,12 +235,18 @@ class AmpBridPlayer extends AMP.BaseElement {
 
     const {partnerID_: partnerID, feedID_: feedID} = this;
 
-    const placeholder = htmlFor(element)`
-      <amp-img referrerpolicy=origin layout=fill placeholder>
-        <amp-img referrerpolicy=origin layout=fill fallback
-            src="https://cdn.brid.tv/live/default/defaultSnapshot.png">
+    const html = htmlFor(element);
+    const placeholder = html`
+      <amp-img referrerpolicy="origin" layout="fill" placeholder>
+        <amp-img
+          referrerpolicy="origin"
+          layout="fill"
+          fallback
+          src="https://cdn.brid.tv/live/default/defaultSnapshot.png"
+        >
         </amp-img>
-      </amp-img>`;
+      </amp-img>
+    `;
```

Comment:
For an example of the utility of this change, see unformatted template with plain color on the removed part of this patch.

There's a subsection `</amp-img>` that was previously highlighted, but that's because GitHub incorrectly interprets it as JSX!